open_fdd/gateway/cli.py

```
@@ -0,0 +1,90 @@
+
+    env_host = (os.environ.get("OFDD_BRIDGE_HOST") or "").strip()
+    if env_host:
+        host = env_host
+    env_port_raw = (os.environ.get("OFDD_BRIDGE_PORT") or "").strip()
+    if env_port_raw:
+        vp = _valid_tcp_port(env_port_raw)
+        if vp is not None:
+            port = vp
+        else:
+            _log.warning(
+                "Ignoring invalid OFDD_BRIDGE_PORT=%r (expected 1–65535); using default %s",
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)